### PR TITLE
core-8350 Save current view (tile / grid) to session and restore previously selected view on next user session

### PR DESCRIPTION
**Pull request by sri2k1us**

### CODE REVIEW

File: de-lib/src/main/java/org/iplantc/de/apps/client/AppsView.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
         AppCategory getSelectedAppCategory();
 
-        void go(HasOneWidget container, HasId selectedAppCategory, HasId selectedApp);
+        void go(HasOneWidget container, HasId selectedAppCategory, HasId selectedApp, String actView);
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/apps/client/AppsView.java]: Maybe we should rename this to `activeView` so it's clear?

File: de-lib/src/main/java/org/iplantc/de/apps/client/presenter/AppsViewPresenterImpl.java [hunk 2]
```diff
@@ -94,13 +95,15 @@ public AppCategory getSelectedAppCategory() {
 
     @Override
     public void go(final HasOneWidget container,
-                   final HasId selectedAppCategory,
-                   final HasId selectedApp) {
+                   final HasId selectedAppCategory, final HasId selectedApp, final String actview) {
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/apps/client/presenter/AppsViewPresenterImpl.java]: These parameters should probably all be on separate lines for consistency. Also, we should probably rename the variable to `activeView` with camelCase.

I think there are also some tests for this class so you might want to check for breakages.

File: de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java [hunk 3]
```diff
@@ -188,6 +189,36 @@ public void loadApps(List<App> apps) {
...
+    }
+
+    /**
+     * Set new active view
+     *
+     * @param newView view to set as active
+     */
+    @Override
+    public void setActiveView(String newView) {
+        if (AppsTileViewImpl.GRID_VIEW.equals(newView)) {
```
Comment (ash3rz) [inline de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java]: I think this should be `AppsListView.GRID_VIEW.equals(newView)` instead?

**Comment by sri2k1us** [comment]
Updated variable names. Tests ran without issues. Thanks for the review!

**Pull request merged**